lib/BankAccount.php

```diff
@@ -14,7 +14,6 @@
  * @property string $country
  * @property string $currency
  * @property string $customer
- * @property bool $default_for_currency
```

Comment:
This property should stay as it still exists: https://stripe.com/docs/api#account_bank_account_object-default_for_currency

Reply:
I thought this object was this: https://stripe.com/docs/api#customer_bank_account_object

What's the difference between these two?

Reply:
The resource represents a Bank Account in the API. This is shared by Customers and Accounts (for External Accounts) since the resource itself has the same shape (except a small number of properties). You can confirm this in the `instance_url` method [here](https://github.com/stripe/stripe-php/blob/master/lib/BankAccount.php#L38-L55) which shows how we calculate the endpoint to use when accessing/updating a bank account